plugins/auth-backend/src/providers/auth0/provider.ts

```diff
@@ -127,6 +133,10 @@ export class Auth0AuthProvider implements OAuthHandlers {
       scope: req.scope,
       state: encodeState(req.state),
       ...(this.audience ? { audience: this.audience } : {}),
+      ...(this.connection ? { connection: this.connection } : {}),
+      ...(this.connectionScope
```

Comment:
How do the `connectionScope` interact with the `req.scope`? passed from the frontend? Are they completely separate things, or are the situations where a user would expect the `req.scope` to be forwarded to `connectionScope`?

Reply:
@Rugvip Auth0 merges the pre-selected scopes with the ones provided in `connection_scope` by default. There is more information about it [here](https://community.auth0.com/t/pass-additional-scopes-permissions-to-a-social-connection/56103).